README.md

```diff
@@ -38,7 +38,7 @@ This command will clone all the class files from the class Git repository into t
 
 ## Making Required Changes
 
-First you should goto the `CS340_sample_flask_app` repository folder and setup a new Python virtual environment for Flask, then install related dependencies. Run the commands within previous directory:
+First you should goto the `CS340_sample_flask_app` repository folder (that is, if it exists... it's possible that the repo files may have just cloned into the directory) and setup a new Python virtual environment for Flask, then install related dependencies. Run the commands within previous directory:
```

Comment:
What do you mean by "just cloned into the directory"? AFAIK `git clone`  would result into a new directory being created.

Reply:
I see that the pull got merged, and I think the important stuff was the run.py and the 'python -m flask run' command.

That being said, I also thought that git clone would create a folder called "CS340_sample_flask_app" in whatever directory it was cloned into, but for some reason I wasn't getting the directory, and there were a few other students who also did not get the directory.  That is, if I create a directory called "flask" and cd to that directory, and then run the git clone, it should create /flask/CS340_sample_flask_app and then all of the repo files should be inside CS340_sample_flask_app.  Interestingly, if I ran a git clone inside of the 'flask' directory, all of the repo files would just be inside the flask directory.

So, instead of /flask/CS340_sample_flask_app/run.py, it would just be /flask/run.py

I have no idea why this is, and in my small sample size of asking around, this particular git clone doesn't actually make a CS340_sample_flask_app folder, at least not on the flip server.  I don't know how prevalent this issue is, so I thought it would be best to leave the references to the CS340_sample_flask_app directory in the directions as some students may have that folder created, but there has been enough confusion about the missing directory that I wanted to make a note of it for future students and let them know that the CS340_sample_flask_app may be missing, that that this is totally fine.

As a test, I'd be curious to know what happens when you (or anyone else who is reading this) run a fresh git clone into a fresh directory on the flip server... does it create the CS340_sample_flask_app directory, or does it just clone the repo files right into whatever directory you cloned into?

thanks!